frontend/app/components/button-link.tsx

```diff
@@ -31,53 +36,88 @@ type ButtonLinkStyleProps = {
 type ButtonLinkProps = ComponentProps<typeof AppLink> &
   ButtonLinkStyleProps & {
     disabled?: boolean;
+    endIcon?: ButtonEndIconProps['icon'];
+    endIconProps?: OmitStrict<ButtonEndIconProps, 'icon'>;
+    startIcon?: ButtonStartIconProps['icon'];
+    startIconProps?: OmitStrict<ButtonStartIconProps, 'icon'>;
   };
 
-export function ButtonLink({
-  children,
-  className,
-  disabled,
-  file,
-  hash,
-  params,
-  pill,
-  search,
-  to,
-  size = 'base',
-  variant = 'default',
-  ...props
-}: ButtonLinkProps) {
-  const baseClassName = cn(
-    'inline-flex items-center justify-center rounded border align-middle font-lato no-underline outline-offset-4',
-    sizes[size],
-    variants[variant],
-    pill && 'rounded-full',
-  );
-
-  if (disabled) {
-    return (
-      <a
-        className={cn(baseClassName, 'pointer-events-none cursor-not-allowed opacity-70', className)}
-        role="link"
-        aria-disabled="true"
-        {...props}
-      >
-        {children}
-      </a>
+/**
+ * Tailwind CSS Buttons from Flowbite
+ * @see https://flowbite.com/docs/components/buttons/
+ *
+ * Disabling a link
+ * @see https://www.scottohara.me/blog/2021/05/28/disabled-links.html
+ */
+const ButtonLink = forwardRef<HTMLAnchorElement, ButtonLinkProps>(
```

Comment:
`forwardRef()` is deprecated in React v19 and will be removed in a future release. Starting with v19, refs are passed as regular props.